man/chpl.txt

```diff
@@ -587,9 +587,6 @@ ENVIRONMENT VARIABLES FOR OPTIONS
   non-empty value of the environment variable equivalent has the same effect
   as passing that option once.
 
-
-ENVIRONMENT
-
   See $CHPL_HOME/doc/chplenv.rst for detailed information about
   general environment variable settings, legal values, and default
   settings. Run $CHPL_HOME/util/printchplenv to view your current
```

Comment:
The text just below this says "Some of the most commonly-used environment variables are summarized here."  I don't think this description works anymore (there are only two and they're not that common).  Maybe a new paragraph saying "There are a few environment variables that do not correlate to flags:"?  

Though even that's not quite accurate since CHPL_DEVELOPER is...  That makes me think that perhaps we should drop CHPL_DEVELOPER from this section and move the description of CHPL_MODULE_PATH into the -M flag's description (?).  Or leave it here as "There is currently one environment variable that doesn't correlate to a command-line flag:"